<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局按钮基础样式：浅灰色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: #f0f0f0; /* 浅灰色背景 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态 */
QPushButton:hover {
    background-color: #e0e0e0; /* 稍深灰色 */
    color: black;
    font-size: 17pt; /* 悬停时字号微涨 */
}

/* 按钮按下状态 */
QPushButton:pressed {
    background-color: #d0d0d0; /* 更深灰色 */
    color: black;
    font-size: 15pt; /* 字号微缩 */
}

/* 功能按钮差异化（灰色基调） */
QPushButton#OpButton {
    background-color: #e8e8e8;
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #f5f5f5;
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #d8d8d8;
    font-size: 19pt;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="3">
       <widget class="QPushButton" name="btnDel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>⌫</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnSqrt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>√</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnClearAll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="btnSign">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>＋/—</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="btnNum0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnPlus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>＋</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnMultiple">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>×</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnDivide">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btnNum1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnNum7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnNum5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnNum9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnNum2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="btnPeriod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnClear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnNum3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnSquare">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>x^2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnMinus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnPercentage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnNum4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btnNum6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnNum8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：白色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: white; /* 基础背景色改为白色 */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #ddd; /* 细边框 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态（保持原有逻辑，背景稍暗） */
QPushButton:hover {
    background-color: #f5f5f5; /* 悬停时浅灰（与白色区分） */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为奶油白（接近#FFFDD0或#FFF8DC等奶油色） */
QPushButton:pressed {
    background-color: #FFF8DC; /* 奶油白背景 */
    color: black;
    font-size: 15pt;
}

/* 功能按钮差异化（保持白色基调，细微区分） */
QPushButton#OpButton {
    background-color: #fafafa; /* 比纯白稍浅灰的底色 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #fcfcfc; /* 接近纯白的浅灰 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #f8f8f8; /* 稍暗于纯白的底色 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="btnEqual">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/* 全局按钮基础样式：浅蓝色背景、无圆角、默认字体、加粗 */
QPushButton {
    background-color: #e6f2ff; /* 浅蓝色背景（柔和浅蓝） */
    font-weight: bold; /* 保留加粗 */
    border: 1px solid #b3d4fc; /* 浅蓝色边框，与背景协调 */
    border-radius: 0; /* 取消圆角 */
    padding: 10px; /* 内边距 */
}

/* 按钮悬停状态 */
QPushButton:hover {
    background-color: #cce5ff; /* 悬停时稍深的浅蓝色 */
    color: black;
    font-size: 17pt;
}

/* 按钮按下状态：改为深蓝色 */
QPushButton:pressed {
    background-color: #4a90e2; /* 深蓝色（突出显点击反馈） */
    color: white; /* 文字变白，与深蓝背景形成对比 */
    font-size: 15pt;
}

/* 功能按钮差异化（蓝色基调，细微区分） */
QPushButton#OpButton {
    background-color: #d1e7fe; /* 比基础色稍浅的蓝 */
    font-size: 18pt;
}

QPushButton#ClearButton {
    background-color: #ebf5ff; /* 接近白色的浅蓝 */
    font-size: 17pt;
}

QPushButton#EqualButton {
    background-color: #bad7f5; /* 稍深于基础色的蓝 */
    font-size: 19pt;
}</string>
        </property>
        <property name="text">
         <string>＝</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnInverse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="display"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
